.gitignore

```diff
@@ -9,8 +9,6 @@ web/assetic
 build/
 vendor/
 vendor
-app/config/parameters.yml
```

Comment:
do we really need this? Besides being hosted, the Sandbox also serves as an example projects for people to check out. It's not that great to commit env specific information in there...